includes/class-evf-smart-tags.php

```diff
@@ -80,6 +80,8 @@ public function process( $content, $form_data, $fields = '', $entry_id = '' ) {
 						} elseif ( in_array( $value['type'], array( 'checkbox', 'payment-checkbox' ), true ) ) {
 							$value = implode( ', ', $value['label'] );
 						}
+					} elseif ( isset( $value['number_of_rating'], $value['value'] ) ) {
+						$value = ( string ) $value['value'] . '/' . ( string ) $value['number_of_rating'];
```

Comment:
@coderkoala Can you setup your VSCode to auto fix linting and beautify the code. Anyway, please adapt `(string)` instead of `( string )`.

Reply:
Resolved with 14e25fe.

And noted, I agree it's being troublesome.